.env.example

```diff
@@ -28,6 +28,16 @@ REDIS_PASSWORD=null
 REDIS_PORT=6379
 
 MAIL_MAILER=smtp
+<<<<<<< HEAD
```

Comment:
Please resolve conflicts the right way 
[Check how to resolve conflicts here](https://stackoverflow.com/questions/38216541/visual-studio-code-how-to-resolve-merge-conflicts-with-git)